test/ButteredBread.t.sol

```diff
@@ -23,6 +28,21 @@ contract ButteredBreadTest is Test {
 
     uint256 public fixedPointPercent;
     address[] public userList;
+    YieldDistributorTestWrapper public yieldDistributor;
+    string public deployConfigPath = string(bytes("./test/test_deploy.json"));
+    string config_data = vm.readFile(deployConfigPath);
+    bytes projectsRaw = stdJson.parseRaw(config_data, "._projects");
+    address[] projects = abi.decode(projectsRaw, (address[]));
+    uint256 _blocktime = stdJson.readUint(config_data, "._blocktime");
+    uint256 _maxPoints = stdJson.readUint(config_data, "._maxPoints");
+    uint256 _precision = stdJson.readUint(config_data, "._precision");
+    uint256 _minVotingAmount = stdJson.readUint(config_data, "._minVotingAmount");
+    uint256 _cycleLength = stdJson.readUint(config_data, "._cycleLength");
+    uint256 _minHoldingDuration = stdJson.readUint(config_data, "._minHoldingDuration");
+    uint256 _lastClaimedBlockNumber = stdJson.readUint(config_data, "._lastClaimedBlockNumber");
+    uint256 _yieldFixedSplitDivisor = stdJson.readUint(config_data, "._yieldFixedSplitDivisor");
+    // See test/YieldDistributor.t.sol for explanation of these values
+    uint256 _minRequiredVotingPower = stdJson.readUint(config_data, "._minRequiredVotingPower");
```

Comment:
i recommend in the future, inheriting the deployment script from the Scripts folder into the test setup, which will also inherit the stdJson datas. but for now, this works